Calibration/HcalAlCaRecoProducers/plugins/AlCaIsoTracksFilter.cc

```diff
@@ -198,141 +198,152 @@ bool AlCaIsoTracksFilter::filter(edm::Event& iEvent, edm::EventSetup const& iSet
 			   << iEvent.bunchCrossing();
 
   //Step1: Find if the event passes one of the chosen triggers
-  trigger::TriggerEvent triggerEvent;
-  edm::Handle<trigger::TriggerEvent> triggerEventHandle;
-  iEvent.getByToken(tok_trigEvt_, triggerEventHandle);
-  if (!triggerEventHandle.isValid()) {
-    edm::LogWarning("HcalIsoTrack") << "Error! Can't get the product "
-				    << triggerEvent_.label() ;
+  bool ok(false);
```

Comment:
@bsunanda for the sake of code readibilty would it be possible to find a variable name more meaningful than "ok"?